README.MD

```diff
@@ -31,6 +121,14 @@ lenni0451.net_39999_c0.30-cpe.viaproxy.127-0-0-1.nip.io
 
 If the domain of a server contains an underscore you currently can't connect to it since ViaProxy uses the underscore as a separator.
 
+#### Geyser - usage for Geyser-ViaProxy
```

Comment:
Why is this section here? This plugin can have uses with Geyser in SRV mode, I don't think this is the place for an explanation how it works...

Reply:
oh, ok, i remove it and leave only GeyserConnect.

Reply:
I was talking about both sections. I don't think the readme of this plugin is a good place to explain the geyser plugin